indexer/packages/postgres/src/db/migrations/migration_files/20240715155120_subaccount_usernames.ts

```diff
@@ -0,0 +1,14 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('subaccount_usernames', (table) => {
+    // username should be unique across the table
+    table.string('username').notNullable().unique();
+    // subaccounts is a foreign key to the subaccounts table subaccounts.id
+    table.uuid('subaccountId').notNullable().primary().references('id').inTable('subaccounts');
```

Comment:
I think we want to support multiple usernames per subaccount at one point right?

Reply:
I thought we landed on 1 username per subaccount and affiliates was going to be handled differently?

Reply:
lets make username primary